core/src/main/kotlin/com/alecstrong/sql/psi/core/postgresql/psi/mixins/ReturningClauseMixin.kt

```diff
@@ -0,0 +1,24 @@
+package com.alecstrong.sql.psi.core.postgresql.psi.mixins
+
+import com.alecstrong.sql.psi.core.ModifiableFileLazy
+import com.alecstrong.sql.psi.core.postgresql.psi.PostgreSqlReturningClause
+import com.alecstrong.sql.psi.core.psi.QueryElement.QueryResult
+import com.alecstrong.sql.psi.core.psi.SqlCompositeElementImpl
+import com.alecstrong.sql.psi.core.psi.SqlResultColumn
+import com.intellij.lang.ASTNode
+import com.intellij.psi.util.PsiTreeUtil
+
+internal abstract class ReturningClauseMixin(node: ASTNode) : SqlCompositeElementImpl(node), PostgreSqlReturningClause {
+
+  private val queryExposed: Collection<QueryResult> by ModifiableFileLazy(containingFile) {
+    listOf(
+      QueryResult(
+        null,
+        PsiTreeUtil.findChildrenOfType(this, SqlResultColumn::class.java)
```

Comment:
Ideally I wanted to call `PostgreSqlReturningClause#getResultColumnList()` but for some reason such a method wasn't generated. In the bnf I'm referencing `result_column` defined in sql.bnf like `{result_column}` which is the only reason that I could see it not working.

Reply:
yea this is a shortcoming of GrammarKitComposer, if you reference elements like `{result_column}` they don't appear in the generated code. Seems like a good one to focus on fixing to make contributing easier

Reply:
https://github.com/AlecStrong/Grammar-Kit-Composer/issues/1